tests/regression/56-witness/48-apron-unassume-no-strengthening.yml

```diff
@@ -0,0 +1,34 @@
+- entry_type: invariant_set
+  metadata:
+    format_version: "2.0"
+    uuid: 9b19e72a-376d-4c34-80d4-f46020de0c92
+    creation_time: 2025-03-12T12:53:06Z
+    producer:
+      name: Simmo Saan
+      version: n/a
```

Comment:
I still think you should provide versions of yourself, e.g., to prepare for the singularity. :wink: